### PR TITLE
Adding version warnings to any versioned doc w/ deprecated Lock/Auth0.js

### PR DESCRIPTION
**Pull request by jeffreylees**
## Passwordless warnings
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/libraries/lock/v9
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/libraries/lock/v10
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/libraries/auth0js/v7
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/libraries/auth0js/v8

## Newly unversioned connections/passwordless docs

- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/spa-sms
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/spa-email-code
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/spa-email-link
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-sms
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-code
- https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-link

Also removes old includes that covered similar ground.

### CODE REVIEW

**Review by mpaktiti — CHANGES_REQUESTED** [review]
The review app does not work https://auth0-docs-content-pr-5709.herokuapp.com/

**Comment by jeffreylees** [comment]
Of course it doesn't :grin:

Will look into it

File: articles/connections/passwordless/regular-web-app-email-code/v8/index.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ You can follow any of the [Regular Web App Quickstarts](/quickstart/webapp) to s
 
 ### Use your own UI
 
+<%= include('../../../../_includes/_version_warning_auth0js') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-email-code/v8/index.md]: `version-warning` can be used only under a version dropdown (that is, at the top of the page).
If the warning is in the middle of the doc you have to use a `warning`

Comment (mpaktiti) [reply articles/connections/passwordless/regular-web-app-email-code/v8/index.md]: This is not the only place so please go through all the files and double check

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Jeff, please list also all the review app URLs I should check. I know that it is troublesome but if you don't do it then I have to search and see what has been updated. Thanks.

**Comment by jeffreylees** [comment]
@mpaktiti 

I fixed the panels - all version pages have version-warnings in the appropriate places now.

I added the links to the connections/passwordless docs to the description.

Each versioned page underneath of Lock v9 and v10 are also included identically, but I assumed you didn't want another twenty or so links -  I can go through and list them if you need them though.

File: articles/connections/passwordless/regular-web-app-email-link/v7/index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: Using Passwordless Authentication with a magic link via email on Regular
 ---
 # Passwordless Authentication with a magic link via e-mail on Regular Web Apps
 
-<%= include('../../_older-version-alert' %>
+<%= include('../../../../_includes/_version_warning_auth0js' %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-email-link/v7/index.md]: The warning says that they should migrate to Auth0.js v9 asap, but [the snippet already uses v9](https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-link/v7#use-your-own-ui)

File: articles/connections/passwordless/regular-web-app-email-code/v7/index.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Using Passwordless Authentication with a one-time code via email on Regul
 ---
 # Passwordless Authentication with a one-time code via e-mail on Regular Web Apps
 
-<%= include('../../_older-version-alert' %>
+<%= include('../../../../_includes/_version_warning_auth0js' %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-email-code/v7/index.md]: Script uses v9 already https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-email-code/v7#use-your-own-ui

File: articles/connections/passwordless/regular-web-app-email-link/v7/index.md (hunk 2 above)
Comment (mpaktiti) [reply articles/connections/passwordless/regular-web-app-email-link/v7/index.md]: I also think this is confusing one way or another. What if someone does not use the custom UI option and he sees this panel telling him to upgrade to Auth0.js v9? For mixed docs (those that include both Lock and Auth0.js) the message should be different (or it should be a `warning` under each respective title)

File: articles/connections/passwordless/regular-web-app-sms/v7/index.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: Using Passwordless Authentication in a Regular Web App with SMS
 ---
 # Authenticate users with a one-time code via SMS in a Regular Web App
 
-<%= include('../../_older-version-alert' %>
+<%= include('../../../../_includes/_version_warning_auth0js' %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-sms/v7/index.md]: script already uses v9 https://auth0-docs-content-pr-5709.herokuapp.com/docs/connections/passwordless/regular-web-app-sms/v7#use-your-own-ui (plus there is a Lock section before that)

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey, I left some comments. I didn't check all of the docs, so please make sure that any adjustments are applied everywhere.

File: articles/connections/passwordless/regular-web-app-email-code.md [hunk 5]
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a one-time code via email on Regul
 ---
 # Passwordless Authentication with a one-time code via e-mail on Regular Web Apps
 
-<%= include('../../_introduction-email', { isMobile: false }) %>
+<%= include('_introduction-email', { isMobile: false }) %>
 
 ## Setup
 
-<%= include('../../_setup-email') %>
+<%= include('_setup-email') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-email-code.md:10]: this is H4, while the `Setup` is H2

File: articles/connections/passwordless/regular-web-app-email-code.md [hunk 6]
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a one-time code via email on Regul
...
 
-<%= include('../../_setup-callback', {spa:false} )%>
+<%= include('_setup-callback', {spa:false} )%>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-email-code.md:12]: same, goes from H2 to H4

File: articles/connections/passwordless/regular-web-app-email-link.md [hunk 7]
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a magic link via email on Regular
 ---
 # Passwordless Authentication with a magic link via e-mail on Regular Web Apps
 
-<%= include('../../_introduction-email-magic-link') %>
+<%= include('_introduction-email-magic-link') %>
 
 ## Setup
 
-<%= include('../../_setup-email') %>
+<%= include('_setup-email') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-email-link.md:10]: goes from H2 to H4

File: articles/connections/passwordless/regular-web-app-email-link.md [hunk 8]
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a magic link via email on Regular
...
 
-<%= include('../../_setup-callback', {spa:false} ) %>
+<%= include('_setup-callback', {spa:false} ) %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-email-link.md:12]: goes from H2 to H4

File: articles/connections/passwordless/regular-web-app-sms.md [hunk 9]
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication in a Regular Web App with SMS
 ---
 # Authenticate users with a one-time code via SMS in a Regular Web App
 
-<%= include('../../_introduction-sms', { isMobile: false }) %>
+<%= include('_introduction-sms', { isMobile: false }) %>
 
 ## Setup
 
-<%= include('../../_setup-sms-twilio') %>
+<%= include('_setup-sms-twilio') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-sms.md:10]: gos from H2 to H4

File: articles/connections/passwordless/regular-web-app-sms.md [hunk 10]
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication in a Regular Web App with SMS
...
 
-<%= include('../../_setup-callback', {spa:false} ) %>
+<%= include('_setup-callback', {spa:false} ) %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/regular-web-app-sms.md:12]: gos from H2 to H4

File: articles/connections/passwordless/spa-email-code.md [hunk 11]
```diff
@@ -4,19 +4,19 @@ description: Learn how to authenticate users with a one-time-code using email in
 ---
 # Using Passwordless Authentication in SPA with Email
 
-<%= include('../../_introduction-email', { isMobile: false }) %>
+<%= include('_introduction-email', { isMobile: false }) %>
 
 ## Setup
 
-<%= include('../../_setup-email') %>
+<%= include('_setup-email') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/spa-email-code.md:11]: gos from H2 to H4

File: articles/connections/passwordless/spa-email-code.md [hunk 12]
```diff
@@ -4,19 +4,19 @@ description: Learn how to authenticate users with a one-time-code using email in
...
 
-<%= include('../../_setup-cors') %>
+<%= include('_setup-cors') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/spa-email-code.md:13]: gos from H2 to H4

File: articles/connections/passwordless/spa-email-link.md [hunk 13]
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a Magic Link via email on SPA
 ---
 # Authenticate users with a Magic Link via e-mail on SPA
 
-<%= include('../../_introduction-email-magic-link') %>
+<%= include('_introduction-email-magic-link') %>
 
 ## Setup
 
-<%= include('../../_setup-email') %>
+<%= include('_setup-email') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/spa-email-link.md:10]: goes from H2 to H4

File: articles/connections/passwordless/spa-email-link.md [hunk 14]
```diff
@@ -3,19 +3,19 @@ title: Using Passwordless Authentication with a Magic Link via email on SPA
...
 
-<%= include('../../_setup-callback', {spa:true} ) %>
+<%= include('_setup-callback', {spa:true} ) %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/spa-email-link.md:12]: goes from H2 to H4

File: articles/connections/passwordless/spa-sms.md [hunk 15]
```diff
@@ -4,19 +4,19 @@ description: Learn how to authenticate users with a one-time-code using SMS in a
 ---
 # Using Passwordless Authentication in SPA with SMS
 
-<%= include('../../_introduction-sms', { isMobile: true }) %>
+<%= include('_introduction-sms', { isMobile: true }) %>
 
 ## Setup
 
-<%= include('../../_setup-sms-twilio') %>
+<%= include('_setup-sms-twilio') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/spa-sms.md:11]: goes from H2 to H4

File: articles/connections/passwordless/spa-sms.md [hunk 16]
```diff
@@ -4,19 +4,19 @@ description: Learn how to authenticate users with a one-time-code using SMS in a
...
 
-<%= include('../../_setup-cors') %>
+<%= include('_setup-cors') %>
```
Comment (mpaktiti) [inline articles/connections/passwordless/spa-sms.md:13]: goes from H2 to H4

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Headers level mess up. Also, since we removed so many docs, can you please add an update feed entry? No need to mention the warnings you added on the other docs, but you should mention the versions that were removed.

**Pull request merged**